src/go/api/scorch/app.go

```diff
@@ -326,6 +332,38 @@ func (this Scorch) stopFilebeat(ctx context.Context, cmd *exec.Cmd, port int) {
 	}
 }
 
+func (this Scorch) recordInfo(runID int, runDir string, name string, startTime time.Time, versionCommit string, versionTag string, versionDate string) error {
```

Comment:
@nblair2 you don't need to pass all the version args here, since you can just use the version package to grab them directly in this function. Also, when two function args are of the same type, you can shorten the arg signature a little bit. In this case, you can do the following:

`recordInfo(runID int, runDir, name string, startTime time.Time) error`